pkg/reconciler/v1alpha1/serverlessservice/serverlessservice_test.go

```diff
@@ -234,6 +251,31 @@ func markNoEndpoints(sks *nv1a1.ServerlessService) {
 	sks.Status.MarkEndpointsNotReady("NoHealthyBackends")
 }
 
+type deploymentOption func(*appsv1.Deployment)
+
+func deploy(namespace, name string, opts ...deploymentOption) *appsv1.Deployment {
```

Comment:
Considering it's 1:1 with the likes in PA, should it be moved to functional.go for reuse?

Reply:
I have mixed feelings about resource creation in `functional.go` since it is at times context sensitive (e.g. `deploy` is different in the Revision tests, `rev` is different in the Route/Service/Configuration tests).  We can consider it, but it feels like we should consider the cleanup of resource creation holistically.